package.json

```diff
@@ -65,6 +65,7 @@
     "rollup": "^2.34.2",
     "tape": "*",
     "ts-node": "^9.0.0",
```

Comment:
Can we delete this ts-node reference as well now?

Reply:
Would be good to tidy up. MRL relies on it at the moment though, so we need to keep it at least at the monorepo level. Will aim to remove (or at least update) in a subsequent PR.